activation-config.yaml

```diff
@@ -56,6 +56,10 @@ googleCloud:
     # Send custom metrics to GCP to diagnose quickly if your dynatrace-gcp-function processes and sends metrics to Dynatrace properly.
     # Allowed values: true/yes, false/no
     selfMonitoringEnabled: false
+    # scopingProjectSupportEnabled: set to true when metrics scope is configured. Metrics will be collected from all monitored projects,
+    # but topology fetch will not, so we need to bypass the verification in case there are instances of a service in monitored projects
```

Comment:
Is this explanation of behavior changes needed here? I can see us giving more tweaks based on this variable and probably we will not remember to update this var description